### PR TITLE
Send title params as the API expects them.

### PR DESCRIPTION
**Pull request by robholland**

### CODE REVIEW

File: clearbit/prospector.py [hunk 1]
```diff
@@ -5,6 +5,8 @@ class Prospector(Resource):
 
     @classmethod
     def search(cls, **options):
+        if options.has_key('titles'):
```
Comment (maccman) [inline clearbit/prospector.py]: I'm not sure this is what we want. Don't we want to do an array check here?

Comment (robholland) [inline clearbit/prospector.py]: We don't need to, either a string or an array will get sent properly. I'll add a test for that.

Comment (maccman) [inline clearbit/prospector.py]: Ok, so an array turns into multiple params? I'd rather not hard-code a property here - can we do an array check and rename the property?

Comment (robholland) [inline clearbit/prospector.py]: We can, but then if they pass a string for titles it will send broken params, which seems a bit harsh?

Comment (maccman) [inline clearbit/prospector.py]: I mean check the property value to see if it's an array or not?

Comment (maccman) [inline clearbit/prospector.py]: https://github.com/maccman/nestful/blob/master/lib/nestful/helpers.rb#L27

Comment (robholland) [inline clearbit/prospector.py]: Updated the PR.

**Comment by maccman** [comment]
lgtm!

**Pull request merged**